app/src/main/java/info/anwesha/iitp/home/MainActivity.java

```diff
@@ -148,14 +150,34 @@ protected void onStart() {
     }
 
     private void populateHeaderView() {
-
+        Calendar c = Calendar.getInstance();
+        int day = c.get(Calendar.DAY_OF_MONTH);
```

Comment:
Expected a better way. But fine, this would work

Reply:
A better approach would have been using a date object of 7th February and then taking subtraction with the current date by using milliseconds!! https://stackoverflow.com/questions/3838527/android-java-date-difference-in-days/3838661

Reply:
OK, will take care.